packages/bcer-data-portal/app/src/components/Navigator.tsx

```diff
@@ -43,11 +43,24 @@ function a11yProps(index: any) {
   };
 }
 
+function setInitial(path: string) {
+  switch (path) {
+    case '/':
```

Comment:
I think to keeps the routes consistent we could use the `routes` object instead of using plain strings.